user_journey/views/event_type.view.lkml

```diff
@@ -0,0 +1,143 @@
+view: event_type {
```

Comment:
Good news! We can share these definitions now, because I figured out how to use the locally referenced filter (see e.g. `{% condition message_id %} event_types.category {% endcondition %}` instead of `{% condition event_type.message_id %} event_types.category {% endcondition %}`).